Tasks/AzureTestPlanV0/task.json

```diff
@@ -13,7 +13,7 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 0,
-    "Minor": 238,
+    "Minor": 237,
```

Comment:
To do: this version needs to be updated to 2.238.2. This keeps reverting on local somehow

Reply:
updated to 2.238.2